scout/server/blueprints/variant/templates/variant/acmg.html

```diff
@@ -42,9 +42,13 @@ <h4>
             <button type="button" id="download-pdf" class="btn btn-secondary btn-sm position-absolute" style="top: 10px; right: 10px;">
               Download as PDF
             </button>
-          {% elif not evaluation %}
-            <button class="btn btn-primary form-control">Submit</button>
+            <br>
           {% endif %}
+            <!-- external links -->
+            Search ClinGen Criteria Specifications (CSPEC):
+            {% for gene in variant.genes %}
+              <a href="{{ gene.cspec_link }}" class="btn btn-secondary text-white" rel="noopener" referrerpolicy="no-referrer" target="_blank">{{ gene.common.hgnc_symbol if gene.common else gene.hgnc_id }}</a>
```

Comment:
I had issues with this link not being populated several times while testing, in particular when browsing to an old classification for "edit" mode. But that is another issue. I think I will just reopen that one, and we can see if we can get it to reproduce properly.